tests/ui/unpretty/expanded-interpolation.rs

```diff
@@ -0,0 +1,99 @@
+//@ compile-flags: -Zunpretty=expanded
```

Comment:
A brief comment at the top of this file explaining what it is testing would be helpful.

Reply:
Added: _"This test covers the AST pretty-printer's insertion of parentheses in some macro metavariable edge cases. Synthetic parentheses (i.e. not appearing in the syntax tree) need to be printed in order for the printed code to be valid Rust syntax. We also test negative cases: the pretty-printer should not be synthesizing parentheses indiscriminately; only where necessary."_